Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -1269,10 +1269,9 @@ static void dispatchSyntheticMouseMove(LocalFrame& mainFrame, const WebCore::Flo
         TextIndicatorOption::IncludeSnapshotWithSelectionHighlight
     };
     if (auto range = std::exchange(m_rangeForDropSnapshot, std::nullopt)) {
-        if (auto textIndicator = TextIndicator::createWithRange(*range, defaultTextIndicatorOptionsForEditDrag, TextIndicatorPresentationTransition::None, { }))
-            textIndicatorData = textIndicator->data();
+        textIndicator = TextIndicator::createWithRange(*range, defaultTextIndicatorOptionsForEditDrag, TextIndicatorPresentationTransition::None, { });
     }
```

Comment:
You can remove the braces around `if`

```cpp
if (auto range = std::exchange(m_rangeForDropSnapshot, std::nullopt))
    textIndicator = TextIndicator::createWithRange(*range, defaultTextIndicatorOptionsForEditDrag, TextIndicatorPresentationTransition::None, { });
```

Reply:
I'm shocked the linter didn't catch this...